pkgs/tools/backup/tarsnapper/default.nix

```diff
@@ -23,6 +23,10 @@ python3Packages.buildPythonApplication rec {
       url = "https://github.com/miracle2k/tarsnapper/commit/2ee33ce748b9bb42d559cc2c0104115732cb4150.patch";
       hash = "sha256-fEXGhzlfB+J5lw1pcsC5Ne7I8UMnDzwyyCx/zm15+fU=";
     })
+    (fetchpatch {
+      url = "https://github.com/miracle2k/tarsnapper/commit/eace01f3085fba8a6421d4f19110b814511e5170.patch?full_index=1";
+      hash = "sha256-2YPb7iaAusT1DkISfOWs72jr/GBY/qG5qFyRlnVt0IY=";
+    })
```

Comment:
This patch is not in the upstream repo. I had a look and it's benign but doing this looks *really* suspicious.

Please either use your namespace to make it obvious that it's a 3rd party patch or include the patch as a text patch. I think I'd prefer the former.

Please also add a comment briefly explaining whether this patch was proposed upstream.

Reply:
Hi, this is a patch that I have done in a pr to the repository. @emilazy said me that it is better to use the upstream repo in the commit url in one of the previous packages where I did the nose to pytest migration

Reply:
Using the origin repo in the URLs is less prone to bitrotting, but yes we need comments linking to PRs here at least. Haven’t had the chance to look at the rest of this yet.

Reply:
If this is indeed from an upstream PR, please link the PR. The comment is not nearly enough to explain the situation.